src/sensor-fusion/fusion-pose-sensor.js

```diff
@@ -139,7 +139,11 @@ FusionPoseSensor.prototype.updateDeviceMotion_ = function(deviceMotion) {
     this.previousTimestampS = timestampS;
     return;
   }
+
   this.accelerometer.set(-accGravity.x, -accGravity.y, -accGravity.z);
+  if (Util.isR7()) {
+    this.gyroscope.set(-rotRate.beta, rotRate.alpha, rotRate.gamma);
+  } else
```

Comment:
small change: curly bracket for the `else` statement and same indentation as the `if`

```
} else {
  this.gyroscope.set(rotRate.alpha, rotRate.beta, rotRate.gamma);
}
```